public/app/features/connections/components/ConnectionsRedirectNotice/ConnectionsRedirectNotice.tsx

```diff
@@ -34,14 +35,14 @@ export function ConnectionsRedirectNotice({ destinationPage }: { destinationPage
   const styles = useStyles2(getStyles);
 
   return (
-    <Alert severity="warning" title="Data sources have a new home!">
+    <Alert severity="warning" title="">
```

Comment:
What is the effect of leaving this title empty? Specially in terms of accessibility.

Reply:
That's a very good question. As the title serves as the `aria-label` for the whole alert, I think ideally we don't want to leave the title out - or provide the aria-label somehow else. 

In this case for example we wanted to leave the title out to make the alert more compact and take up less height and don't push the original content down that much.

I am not sure how to do this easily to be honest, as there can be cases where we don't want the title on the UI, however we would still like to have an `aria-label` for the sake of accessibility. Without changing the props of the Alert component maybe we could add it on the call-site to the element that wraps the Alert component?

Reply:
There are ways you can set an aria label for an element from other element. See `aria-labelledby`. Generally speaking we should not leave this without an aria label.

Reply:
Yepp, above I meant `aria-labelledby`, that's what we are using in Alert today. The problem is that if there is no title, then the referenced element is empty. I am not really sure how to support this design from an accessibility perspective, the only way I can imagine would be to introduce a new prop called something like `hideTitle`, but I think that would be a bit too much.

Reply:
There is also an option to keep the whole text in the title and just position the <LinkButton.../> absolutely to solve the padding issue outside of the Alert component...

Reply:
**Update 2.0:** I have decided to move most of the `<Alert>` related changes to a separate PR so they are easier to be looked at: https://github.com/grafana/grafana/pull/61242